api/index/indexRouter.js

```diff
@@ -33,3 +33,5 @@ router.get('/', function (req, res) {
 });
 
 module.exports = router;
+
+
```

Comment:
This file probably doesn't have to be included in the PR

Reply:
Good Catch, Thanks!